README.md

```diff
@@ -52,6 +52,10 @@ To contribute, check out our [issue tracker](https://github.com/localstack/local
     $ make hot-reloading
     ```
 
+4. Disable hot reloading:
```

Comment:
I expected more around this - something that explains why I have to disable hot reloading manually.